homework03/slices/slices1.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "fmt"
+
+func main() {
+
+	s := []string{"one", "two", "three"}
+	s1 := []string{"one", "two"}
+	fmt.Println(max(s))
+	fmt.Println(max(s1))
+}
+
+func max(s []string) string {
+
+	var i, length, maxLen int
+	longest := s[i]
+	for _, word := range s {
+		length = len(word)
+
+		if length > maxLen {
```

Comment:
`maxLen` never changes. Your code works because in your function call you use `s := []string{"one", "two", "three"}` parameter, where the values are already in correct order. Try to use this input: `s := []string{"aaa", "bb", "c"}` and see the result. Change this please